lib/tessel/wifi.js

```diff
@@ -158,7 +158,7 @@ Tessel.prototype.setWiFiState = function(enable) {
             // End the connection
             remoteProcess.close();
           }, Tessel.__wifiConnectionTimeout);
-          remoteProcess.stdout.once('data', function(data) {
+          remoteProcess.stdout.on('data', function(data) {
```

Comment:
I found an issue where sometimes the first data back from this command doesn't contain the exact substring we're searching for. In that case, we should keep listening and not reject immediately.